app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,56 @@
+class TasksController < ApplicationController
+  before_action :set_project
+  before_action :set_task, only: [:show, :edit, :update, :destroy]
```

Comment:
Можно данную строчку красиво организовать:
```suggestion
  before_action :set_task, only: %i[show edit update destroy]
```
Загугли, что это такое, в RoR-разработке постоянно используется и приветствуется